templates/ts/sso-tab-with-obo-flow/cli.md

```diff
@@ -0,0 +1,19 @@
+## Try sample with TeamsFx CLI
```

Comment:
Usually, we don't include cli.md for template because user has already created the tab app.